### ISSUE TITLE
Data import fails when more than one channel is being used

### ISSUE THREAD

**Issue by mh-xdev** [microstream-one/microstream#254]
## Environment Details
* MicroStream Version: 05.00.02-MS-GA
* JDK version: 11
* OS: Windows 10 20H2 (Build 19042.1237)
* Used frameworks: Micronaut

## Describe the bug
I've tried to import data using the Data [Import / Export feature](https://docs.microstream.one/manual/storage/import-export.html) into a MicroStream database inside a application that uses the Micronaut framework. However, I've noticed that this will inevitably fail with an exception as long as you're using more than one channel. 

## To Reproduce
1) Create a MicroStream database with more than one channel
2) Export data 
3) Clear data (not sure if this is required)
4) Import the data you've recieved from Step 2

## Expected behavior
A successful import of the data, regardless of the number of channels being used.

## Additional context

Stack trace:
```←[36m09:54:05.752←[0;39m ←[1;30m[default-nioEventLoopGroup-1-2]←[0;39m ←[1;31mERROR←[0;39m ←[35mi.m.http.server.RouteExecutor←[0;39m - Unexpected error occurred: Problem in channel 1
one.microstream.storage.exceptions.StorageException: Problem in channel 1
        at one.microstream.storage.types.StorageChannelTask$Abstract.checkForProblems(StorageChannelTask.java:84)
        at one.microstream.storage.types.StorageChannelTask$Abstract.waitOnCompletion(StorageChannelTask.java:143)
        at one.microstream.storage.types.StorageRequestAcceptor$Default.waitOnTask(StorageRequestAcceptor.java:159)
        at one.microstream.storage.types.StorageRequestAcceptor$Default.importFiles(StorageRequestAcceptor.java:256)
        at one.microstream.storage.types.StorageConnection$Default.importFiles(StorageConnection.java:583)
        at one.microstream.training.export.controller.BookController.importBinary(BookController.java:100)
        at one.microstream.training.export.controller.$BookController$Definition$Exec.dispatch(Unknown Source)
        at io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invoke(AbstractExecutableMethodsDefinition.java:338)
        at io.micronaut.context.DefaultBeanContext$4.invoke(DefaultBeanContext.java:503)
        at io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:246)
        at io.micronaut.web.router.RouteMatch.execute(RouteMatch.java:111)
        at io.micronaut.http.server.RouteExecutor.lambda$executeRoute$15(RouteExecutor.java:605)
        at reactor.core.publisher.FluxDefer.subscribe(FluxDefer.java:46)
        at reactor.core.publisher.Flux.subscribe(Flux.java:8402)
        at reactor.core.publisher.FluxFlatMap.trySubscribeScalarMap(FluxFlatMap.java:200)
        at reactor.core.publisher.FluxFlatMap.subscribeOrReturn(FluxFlatMap.java:93)
        at reactor.core.publisher.Flux.subscribe(Flux.java:8388)
        at io.micronaut.http.server.netty.RoutingInBoundHandler.handleRouteMatch(RoutingInBoundHandler.java:574)
        at io.micronaut.http.server.netty.RoutingInBoundHandler.channelRead0(RoutingInBoundHandler.java:439)
        at io.micronaut.http.server.netty.RoutingInBoundHandler.channelRead0(RoutingInBoundHandler.java:139)
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:102)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.micronaut.http.netty.reactive.HandlerPublisher.channelRead(HandlerPublisher.java:351)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.micronaut.http.netty.stream.HttpStreamsHandler.channelRead(HttpStreamsHandler.java:266)
        at io.micronaut.http.netty.stream.HttpStreamsServerHandler.channelRead(HttpStreamsServerHandler.java:123)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
        at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
        at io.netty.handler.codec.http.HttpServerKeepAliveHandler.channelRead(HttpServerKeepAliveHandler.java:64)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.flow.FlowControlHandler.dequeue(FlowControlHandler.java:200)
        at io.netty.handler.flow.FlowControlHandler.channelRead(FlowControlHandler.java:162)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
        at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
        at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:311)
        at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:432)
        at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:276)
        at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
        at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
        at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
        at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
        at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:719)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:655)
        at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:581)
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: one.microstream.storage.exceptions.StorageException: null
        at one.microstream.storage.types.StorageFile$Abstract.internalOpenWriting(StorageFile.java:462)
        at one.microstream.storage.types.StorageFile$Abstract.ensureWritable(StorageFile.java:414)
        at one.microstream.storage.types.StorageFile$Abstract.ensureReadable(StorageFile.java:404)
        at one.microstream.storage.types.StorageFile$Abstract.copyTo(StorageFile.java:283)
        at one.microstream.storage.types.StorageFileWriter.writeImport(StorageFileWriter.java:76)
        at one.microstream.storage.types.StorageFileManager$Default.importBatch(StorageFileManager.java:1676)
        at one.microstream.storage.types.StorageFileManager$Default$ImportHelper.accept(StorageFileManager.java:1737)
        at one.microstream.storage.types.StorageFileManager$Default$ImportHelper.accept(StorageFileManager.java:1)
        at one.microstream.storage.types.StorageRequestTaskImportData$SourceFileSlice.iterateBatches(StorageRequestTaskImportData.java:522)
        at one.microstream.storage.types.StorageFileManager$Default.copyData(StorageFileManager.java:1626)
        at one.microstream.storage.types.StorageChannel$Default.importData(StorageChannel.java:637)
        at one.microstream.storage.types.StorageRequestTaskImportData$Default.internalProcessBy(StorageRequestTaskImportData.java:384)
        at one.microstream.storage.types.StorageRequestTaskImportData$Default.internalProcessBy(StorageRequestTaskImportData.java:1)
        at one.microstream.storage.types.StorageChannelTask$Abstract.processBy(StorageChannelTask.java:220)
        at one.microstream.storage.types.StorageChannel$Default.work(StorageChannel.java:408)
        at one.microstream.storage.types.StorageChannel$Default.run(StorageChannel.java:495)
        ... 1 common frames omitted
Caused by: one.microstream.afs.exceptions.AfsExceptionExclusiveAttemptConflict: File is already used by a different exclusive user: one.microstream.afs.types.AFile$Default@e007c29("export-dir/java.lang.String_30.bin"). Exclusive user: (java.lang.Thread@4767a185) Thread[MicroStream-StorageChannel-0,5,main]. Attempting user: (java.lang.Thread@4c692cdb) Thread[MicroStream-StorageChannel-1,5,main].
        at one.microstream.afs.types.AccessManager$Default$2.handleExclusiveAttemptConflict(AccessManager.java:732)
        at one.microstream.afs.types.AccessManager$Default.internalUseWriting(AccessManager.java:641)
        at one.microstream.afs.types.AccessManager$Default.useWriting(AccessManager.java:482)
        at one.microstream.afs.types.AccessManager.useWriting(AccessManager.java:92)
        at one.microstream.afs.types.AFile.useWriting(AFile.java:118)
        at one.microstream.storage.types.StorageFile$Abstract.internalOpenWriting(StorageFile.java:444)
        ... 16 common frames omitted
```

Export code:
```
		NioFileSystem fileSystem = NioFileSystem.New();
		String fileSuffix = "bin";
		
		StorageConnection connection = DB.storageManager.createConnection();
		
		StorageEntityTypeExportStatistics exportResult    = connection.exportTypes(
			new StorageEntityTypeExportFileProvider.Default(
				fileSystem.ensureDirectoryPath("export-dir"),
				fileSuffix
			),
			typeHandler -> true
		);
		
		XSequence<Path> exportFiles = CQL
			.from(exportResult.typeStatistics().values())
			.project(s -> Paths.get(s.file().identifier()))
			.execute()
		;
```

Import code:
```
		NioFileSystem          fileSystem = NioFileSystem.New();

		StorageConnection      connection = DB.storageManager.createConnection();
		
		connection.importFiles(X.Enum(fileSystem.ensureDirectoryPath("export-dir").listFiles()));
```

Configuration (as XML):
```
<properties>
    <property name="storage-directory" value ="data" />
    <property name="channel-count" value ="4" />
</properties>
```

**Comment by hg-ms** [comment]
Fix available in PR #262

**Comment by hg-ms** [comment]
Fixed in version 06.00.00-MS-GA
